tools/test/devicesim/Dockerfile

```diff
@@ -37,13 +37,16 @@ RUN mkdir -p $GOPATH \
       github.com/google/gnxi/gnmi_capabilities \
       github.com/google/gnxi/gnmi_get \
       github.com/google/gnxi/gnmi_set \
-      github.com/google/gnxi/gnmi_target
+      github.com/google/gnxi/gnmi_target \
+      github.com/openconfig/gnmi/cmd/gnmi_cli
```

Comment:
I would agree with Sean here. even though the gNMI_cli is important here I think we do not want it deployed in the emulator docker container.